src/components/tables/TableConnected.tsx

```diff
@@ -77,14 +106,25 @@ const mapDispatchToProps = (
     onUnmount: () => {
         dispatch(removeTable(ownProps.id));
     },
-    onModifyData: (shouldResetPage: boolean, tableCompositeState: ITableCompositeState, previousTableCompositeState: ITableCompositeState) => {
+    onWillUpdate: (actions: IActionOptions[]) => {
+        if (actions.length) {
+            dispatch(addActionsToActionBar(`${ownProps.id}action-bar`, actions));
+        }

```

Comment:
Shouldn't we dispatch `addActionsToActionBar(`${ownProps.id}action-bar`, [])` when it's empty to remove the actions?

Reply:
use `getTableChildComponentId(ownProps.id, TableChildComponent.ACTION_BAR)` to get the action-bar id